docs/tracing.md

```diff
@@ -33,16 +33,35 @@ outputting a `Trace` level log when entering and exiting every function.
 It is disabled by default at compile-time. Tracing functionality has no
 impact on the release binary.
 
+You can use `cargo run --bin clippy-tracing --` to build and run the
+latest version in the repo or you can run
+`cargo install --path src/clippy-tracing` to install the binary then use
+`clippy-tracing` to run this binary.
+
+You can run `clippy-tracing --help` for help.
+
 To enable tracing in Firecracker, add instrumentation with:
 
 ```
 clippy-tracing \
 --action fix \
 --path ./src \
---exclude vmm_config/logger.rs,virtio/gen,bindings.rs,net/gen,benches,logger/,signal_handler.rs,time.rs
+--exclude \
+benches,\
+virtio/gen,bindings.rs,net/gen,\
+log-instrument-macros/,log-instrument/,clippy-tracing/,\
+vmm_config/logger.rs,logger/,signal_handler.rs,time.rs
 ```
 
-and re-compile with `--features tracing`:
+`--exclude` can be used to avoid adding instrumentation to specific
+files, here it is used to avoid adding instrumentation in:
+
+- tests.
```

Comment:
do we need full stops after every item?

Reply:
I use bullet points as if they could be separate sentences.

So:
```
it is used to avoid adding instrumentation in tests.
it is used to avoid adding instrumentation in bindings.
it is used to avoid adding instrumentation in the instrumentation tooling.
```
becomes:
```
it is used to avoid adding instrumentation in [tests|bindings|the instrumentation tooling].
```
which becomes:
```
it is used to avoid adding instrumentation in:
- tests.
- bindings.
- the instrumentation tooling.
```
This makes the most sense to me, although I do not know a reference for this.